### ISSUE TITLE
fails after php tag

### ISSUE THREAD

**Issue by torisanbird** [squizlabs/PHP_CodeSniffer#3652]
**Describe the bug**

Some files have errors and cannot be repaired, so the code has been truncated to make it shorter and easier to understand.

I don't know if it's a bug or not.
But other tools don't fail.

phpcbf --version
PHP_CodeSniffer version 3.7.1 (stable) by Squiz (http://www.squiz.net)

**Code sample**
```php
<?php

function test()
{
?> <?php

printf("%d",
1);
}

```

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs test.php ...`
3. See error message displayed

PHPCS output here
```
C:\tmp>phpcbf --standard=PSR2 -v text2.php
Registering sniffs in the PSR2 standard... DONE (43 sniffs registered)
Creating file list... DONE (1 files in queue)
Changing into directory C:\tmp
Processing text2.php [PHP => 28 tokens in 9 lines]... DONE in 24ms (4 fixable violations)
        => Fixing file: 1/4 violations remaining [made 50 passes]... ERROR in 551ms

PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
C:\tmp\text2.php                                      FAILED TO FIX
----------------------------------------------------------------------
A TOTAL OF 3 ERRORS WERE FIXED IN 1 FILE
----------------------------------------------------------------------
PHPCBF FAILED TO FIX 1 FILE
----------------------------------------------------------------------

Time: 826ms; Memory: 8MB

```

**Comment by jrfnl** [comment]
Confirmed as reproducable.

Looks like a fixer conflict between the `Generic.WhiteSpace.ScopeIndent` sniff and the `PEAR.Functions.FunctionCallSignature` sniff.
